user-interface/tsconfig.json

```diff
@@ -1,6 +1,6 @@
 {
   "compilerOptions": {
-    "target": "es5",
+    "target": "ESNext",
```

Comment:
Check into this to make sure this doesn't become a problem if es is upgraded beyond what vite can handle.